TTS/trainer.py

```diff
@@ -484,14 +495,14 @@ def _optimize(
                 with amp.scale_loss(loss_dict["loss"], optimizer) as scaled_loss:
                     scaled_loss.backward()
                 grad_norm = torch.nn.utils.clip_grad_norm_(
-                    amp.master_params(optimizer), grad_clip, error_if_nonfinite=False
```

Comment:
This parameter is not present in previous pytorch versions and the default is False. To maintain compatibility with pytorch previous versions I removed it.